docs/console/auth/authentication.md

```diff
@@ -3,7 +3,7 @@ title: Authentication
 description: Configure and deploy authentication for an app
```

Comment:
Change the name of this page to Getting started

Reply:
Done!